src/main/java/org/jabref/logic/util/BuildInfo.java

```diff
@@ -16,14 +16,14 @@
     public static final String OS_ARCH = System.getProperty("os.arch", UNKNOWN_VERSION).toLowerCase(Locale.ROOT);
     public static final String JAVA_VERSION = System.getProperty("java.version", UNKNOWN_VERSION).toLowerCase(Locale.ROOT);
 
-    private final Version version;
-    private final String authors;
-    private final String developers;
-    private final String year;
-    private final String azureInstrumentationKey;
-    private final String minRequiredJavaVersion;
-    private final boolean allowJava9;
-
+    public final Version version;
```

Comment:
Just out of interest: I thought it Java convention to always hide fields with getters. In which cases is it ok to have public variables?

Reply:
It used to be a convention which was even more manifested due to libraries depending on the existence on getters/setters. In such a case with immutable variables, adding `public` getters is just useless since you are not hiding anything.